src/settings.py

```diff
@@ -30,6 +34,12 @@ class Settings(BaseSettings):
     uvicorn_root_path: str = ""
     uvicorn_proxy_headers: bool = True
 
+    # processing server settings
+    storage_server_ip: str = ""
```

Comment:
We may want to use a non-standard SSH port if we have this public. There are many scanners on 22 with automated dictionary attacks out there.

Reply:
We are currently using public key authentication, not password based. I dont know if it is enabled.